cmd/commands/namespace.go

```diff
@@ -70,6 +71,7 @@ func NamespaceInit(manifests map[string]*core.Manifest, kc *core.KubectlClient)
 
 	command.Flags().StringVarP(&options.Manifest, "manifest", "m", "stable", "manifest of YAML files to be applied; can be a named manifest (stable or latest) or a path or URL of a manifest file")
 
+	command.Flags().BoolVarP(&options.NoSecret, "no-secret", "n", false, "no secret required (unauthenticated)")
```

Comment:
could this read "no secret required for the image registry (unauthenticated)"

Reply:
Yes. How about dropping "(unauthenticated)" and saying simply "no secret required for the registry".

Reply:
perfect

Reply:
I wouldn't use `n` as a shorthand, as that value is used elsewhere for `--namespace`. I think all shorthands should be used consistently accross commands if possible. Obviously namespace as a flag doesn't make sense here, but still

Reply:
probably doesn't need to have a shorthand

Reply:
I removed the shorthand.